responsibleai/tabular/components/src/rai_component_utilities.py

```diff
@@ -130,22 +147,52 @@ def load_mltable(mltable_path: str) -> pd.DataFrame:
         tbl = mltable.load(mltable_path)
         df: pd.DataFrame = tbl.to_pandas_dataframe()
     except Exception as e:
-        _logger.info("Failed to load MLTable")
-        _logger.info(e)
+        _logger.info(f"Failed to load {mltable_path} as MLTable. ")
+        raise e
```

Comment:
are we sure we should raise here?  The other changes look like whitespace but this one seems more serious.

Reply:
this is from Kin's PR https://github.com/Azure/RAI-vNext-Preview/pull/162/files